exec/java-exec/src/main/java/org/apache/drill/exec/work/metadata/ServerMetaProvider.java

```diff
@@ -76,7 +76,7 @@
       .setReadOnly(false)
       .setGroupBySupport(GroupBySupport.GB_UNRELATED)
       .setLikeEscapeClauseSupported(true)
-      .setNullCollation(NullCollation.NC_AT_END)
+      .setNullCollation(NullCollation.NC_HIGH)
```

Comment:
My understanding is NULL collation should be specified together with ASC/DESC.  ASC/NULL LAST as the default option essentially implies NC_HIGH. However, I'm not sure if we just specify NC_HIGH alone, or ASC/NC_AT_END combined.